libnavigation-core/src/main/java/com/mapbox/navigation/core/history/model/HistoryEventSetRoute.kt

```diff
@@ -0,0 +1,63 @@
+package com.mapbox.navigation.core.history.model
+
+import com.mapbox.api.directions.v5.DirectionsCriteria
+import com.mapbox.api.directions.v5.models.DirectionsResponse
+
+/**
+ * @param directionsResponse the route response used
+ */
+class HistoryEventSetRoute internal constructor(
+    val directionsResponse: DirectionsResponse,
```

Comment:
Why is it a directions response and not a directions route? We won't have access to any response data over here anyway (like codes, messages, response waypoints, etc).

Reply:
yeah, supposed to be `DirectionsRoute`.  native's `SetRouteHistoryRecord` calls it `routeResponse`

the `MapboxHistoryTest` instrumentation test proves this works with nav-native